src/status_im/ui/screens/privacy_and_security_settings/delete_profile.cljs

```diff
@@ -0,0 +1,86 @@
+(ns status-im.ui.screens.privacy-and-security-settings.delete-profile
+  (:require [status-im.ui.components.react :as react]
+            [quo.core :as quo]
+            [status-im.multiaccounts.core :as multiaccounts]
+            [status-im.ui.components.chat-icon.screen :as chat-icon.screen]
+            [re-frame.core :as re-frame]
+            [status-im.ui.components.topbar :as topbar]
+            [status-im.ui.components.styles :as components.styles]
+            [status-im.i18n :as i18n]
+            [reagent.core :as reagent]
+            [status-im.utils.security :as security]
+            [status-im.ui.screens.privacy-and-security-settings.events :as delete-profile]))
+
+(defn valid-password? [password]
+  (let [data (security/safe-unmask-data password)]
```

Comment:
Actually now you can also count without unmasking of data https://github.com/status-im/status-react/blob/develop/src/status_im/utils/security.cljs#L15-L17

Reply:
thanks